spec/features/filter_links_by_tag_spec.rb

```diff
@@ -0,0 +1,21 @@
+# feature  'Filtering links' do
+#
+# 	before(:each) do
+# 		Link.create(url: 'http://www.makersacademy.com', message: 'Makers Academy', tags: [Tag.first_or_create(name: 'education')])
+# 		Link.create(url: 'http://www.google.com', message: 'Google', tags: [Tag.first_or_create(name: 'search')])
```

Comment:
Metrics/LineLength: Line is too long. [109/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)